8_child_illness.do

```diff
@@ -68,10 +68,10 @@ order *,sequential  //make sure variables are in order.
         gen c_diarrhea_mof = (h38 == 5) if !inlist(h38,.,8) & c_diarrhea == 1
 
 *c_diarrhea_medfor Get formal medicine except (ors hmf home other_med, country specific). 
-        egen medfor = rowtotal(h12z h15 h15a h15b h15c h15e h15g h15h h15i h15j h15k h15l),mi
+        egen medfor = rowtotal(h12z h15 h15a h15b h15c h15e h15h h15j h15k h15l),mi
 		gen c_diarrhea_medfor = ( medfor > = 1 ) if c_diarrhea == 1
 		// formal medicine don't include "home remedy, herbal medicine and other"
-        replace c_diarrhea_medfor = . if (h12z == 8 | h15 == 8 | h15a == 8 | h15b == 8 | h15c == 8 | h15e == 8  | h15g == 8 | h15h == 8 | h15i == 8 | h15j == 8 | h15k == 8 | h15l == 8)                                       
+        replace c_diarrhea_medfor = . if (h12z == 8 | h15 == 8 | h15a == 8 | h15b == 8 | h15c == 8 | h15e == 8  | h15h == 8 | h15j == 8 | h15k == 8 | h15l == 8)                                       
```

Comment:
This set of code is for all surveys in Recode V. If you find a survey-specific situation, please add using
`if inlist(name,""){}` instead of changing the common code.
As for "unknown injection" & "unknown pill or syrup", to be honest, I have never doubt that they are medfor, thus I never double check with others. Please allow me to.